package/boot/uboot-envtools/files/ramips

```diff
@@ -114,6 +114,11 @@ snr,cpe-w4n-mt)
 	[ -n "$idx" ] && \
 		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x1000" "0x1000"
 	;;
+tplink,deco-x20-v3)
+	idx="$(find_mtd_index ubootenv)"
+	[ -n "$idx" ] && \
+		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x1000" "0x100000"
```

Comment:
The final parameter is block size. For nand flash, usually it is `0x20000` 128 KiB.

Reply:
I was trying a few variations of the block size, and it would end up with CRC errors in uboot, I will test again, but what is currently present seems to "pass" CRC.

Reply:
OK, 0x20000 works correctly without CRC errors, thanks!